src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin/Microsoft.Commerce/UsageConnections/GetUsageConnection.cs

```diff
@@ -25,6 +25,7 @@ namespace Microsoft.AzureStack.Commands
     /// </summary>
     [Cmdlet(VerbsCommon.Get, Nouns.UsageConnection)]
     [OutputType(typeof(UsageConnectionModel))]
+    [Alias("Get-AzureRMUsageConnection")]
```

Comment:
Rm  PowerShell cmdlets are not case senitive, but we want to use the correct cpaitalization in the alias